bitnami/wordpress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
-apiVersion: v1
+apiVersion: v2
 name: wordpress
-version: 9.5.6
+version: 10.0.0
```

Comment:
I think you're right a MAJOR version bump is appropriate, since this is an API backwards-incompatible change.

In keeping with this, I agree with @juan131 you'll also want to add a `To 10.0.0`  section under [Upgrading](https://github.com/bitnami/charts/blob/master/bitnami/wordpress/README.md#upgrading) in the README, noting the drop of Helm 2 support.